lua/ruby_nvim/cmd.lua

```diff
@@ -89,4 +83,31 @@ M.test = function(current_line)
   run_in_floating_window(opts.load("test_cmd"), args)
 end
 
+-- :RubyBrowseGem
+
```

Comment:
For consistency, we might delete this line (there's no blank line in the other command definitions).

Reply:
Ok